Implement saveFile for Web (#1715)
miguelpruivo/flutter_file_picker: CHANGELOG.md

## 9.1.0
### Web

> - `saveFile()` now downloads the file. See the note in the updated [wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki/api#-savefile)

Reviewer: ```suggestion
- Added an implementation for `saveFile()` on the web. See the updated [wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki/api#-savefile)
```

> - `saveFile()` now downloads the file. See the note in the updated [wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki/api#-savefile)

Reviewer: What change do I need to do on the Wiki for this?

Author: Simply indicate that the web platform is supported and what it produces is the download of the file.

> - Added an implementation for `saveFile()` on the web. See the updated [wiki](https://github.com/miguelpruivo/flutter_file_picker/wiki/api#-savefile)

Reviewer: I'll update the wiki separately, no need to mention that here.

```suggestion
- Added an implementation for `saveFile()` on the web.
```
